assets/js/base/components/store-notices-container/style.scss

```diff
@@ -27,8 +27,9 @@
 
 .components-snackbar-list {
 	position: fixed;
-	bottom: 100px;
-	left: 100px;
+	bottom: 40px;
+	left: 16px;
+	width: auto;
```

Comment:
One more comment: I think after adding `width: auto` here, it should be safe to remove L39 `width: fit-content;`.